src/index.html

```diff
@@ -5,9 +5,41 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__image-block">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Card image"
+        >
+      </div>
+      <div class="card__info">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        <div class="card__product-code">
+          Product code: 195434
+        </div>
+        <div class="stars card__stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="stars__text">Reviews: 5</span>
+        </div>
+        <div class="card__price-block">
+          <span class="card__price-text">Price:</span>
+          <span class="card__price-value">$2,199</span>
+        </div>
+        <button class="card__button" data-qa="hover">Buy</button>
+      </div>
+    </div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML We need it to add some "air" and simplify reading. **But don't add them between parent and child elements.**

![image](https://github.com/mate-academy/layout_product-cards/assets/96138503/3b7666c1-26d2-4d0b-9c92-9d8d5352b5fc)

https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules